packages/web-runtime/src/store/user.ts

```diff
@@ -51,6 +51,12 @@ const mutations = {
     state.capabilities = data.capabilities
     state.version = data.version
   },
+  /**
+   * Legacy for oC10, in OCIS quota sticks on the respective drive
```

Comment:
FYI was thinking with @dschmidt today that we might want to introduce a virtual personal drive for oc10 compatibility... nothing for this PR though

Reply:
Would be nice 💪😊